core/src/test/scala/unit/kafka/server/ConsumerGroupHeartbeatRequestTest.scala

```diff
@@ -54,18 +58,92 @@ class ConsumerGroupHeartbeatRequestTest(cluster: ClusterInstance) {
     assertEquals(expectedResponse, consumerGroupHeartbeatResponse.data)
   }
 
-  @ClusterTest(serverProperties = Array(
+  @ClusterTest(clusterType = Type.KRAFT, serverProperties = Array(
```

Comment:
will we have a full integration test for the new group coordinator?

Reply:
Yeah, we will. We have to do two things: 1) parameterise existing integration/system tests to run with the new group coordinator; and 2) extend integration/system tests where needed.

Reply:
Hmmm does this also mean that we are no longer testing the old group coordinator (at least on this test)

Reply:
I guess we will do many of these test changes in a followup though?

Reply:
Correct.